spec-main/fixtures/pages/overlay.html

```diff
@@ -3,6 +3,14 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width">
+    <script>
+      document.title = document.title;
```

Comment:
this is seemingly a no-op, could we at least have a comment here for a future test spelunker?